devicetypes/technisat/technisat-roller-shutter-switch.src/technisat-roller-shutter-switch.groovy

```diff
@@ -0,0 +1,432 @@
+/**
+ *  Copyright 2021 TechniSat
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+
+metadata {
+	definition (name: "TechniSat Roller shutter switch", namespace: "TechniSat", author: "TechniSat", vid:"generic-shade",
+				mnmn: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012',
+				executeCommandsLocally: false) {
```

Comment:
This `vid` doesn't support Power Meter and Energy Meter.
You can create your own presentation using SmartThings CLI.
Please see https://smartthings.developer.samsung.com/docs/devices/device-profile.html#Device-Presentation and https://github.com/SmartThingsCommunity/smartthings-cli#smartthings-capabilitiespresentation-id-version